packages/components/src/tabs/types.ts

```diff
@@ -105,7 +105,6 @@ export type TabListProps = {
 };
 
 // TODO: consider prop name changes (tabId, selectedTabId)
-// switch to auto-generated README
 // compound technique
```

Comment:
@ciampo Not sure if these TODOs were even meant to be committed 😂 Do you want me to remove the rest, too?